src/web/mjs/connectors/Manatoki.mjs

```diff
@@ -0,0 +1,52 @@
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
+
+export default class Manatoki extends GnuBoard5BootstrapBasic2 {
+
+    constructor() {
+        super();
+        super.id = 'manatoki82';
+        super.label = 'Manatoki';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://manatoki82.net';
+
+        this.path = '/comic/p%PAGE%';
+        this.pathMatch = /comic\/p(\d+)/;
+        this.queryMangasPageCount = 'div.list-page ul.pagination li:last-child a';
+        this.queryMangas = 'ul#webtoon-list-all li div.img-item div.in-lable a';
+        this.queryChapter = 'div.serial-list li.list-item div.wr-subject a';
+        this.scriptPages = `
+        new Promise(resolve => {
+            resolve([...document.querySelectorAll('div.view-padding div > img')].map(image => JSON.stringify(image.dataset)));
+        });
+        `;
+
+    }
+
+    async _getPages(chapter) {
+        let request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        return (await Engine.Request.fetchUI(request, this.scriptPages)).map(img=> img.match(/"\S{11}":"(.*)"/)[1]);
```

Comment:
May embed this logic into the `scriptPages`, so the `_getPages` method must not be overwritten.

Reply:
i know this is possible but 
            resolve([...document.querySelectorAll('div.view-padding div > img')].map(image => JSON.stringify(image.dataset).match(/"\S{11}":"(.*)"/)[1]));

didn't work

Reply:
You need to double escape the regex escape characters since the regex is defined within the script string.
Try `/"\\S{11}":"(.*)"/`

Reply:
To test your `script` go to the website such as https://manatoki82.net/comic/93125, then open developer tools and inject your script with `eval` into the console, e.g.
```javaacript
eval(`
    new Promise(resolve => {
        resolve([...document.querySelectorAll('div.view-padding div > img')].map(image => JSON.stringify(image.dataset).match(/"\\S{11}":"(.*)"/)[1]));
    });
`);
```

Reply:
i can't figure out how to do this

Reply:
![image](https://user-images.githubusercontent.com/27136773/96916506-ed675c00-14a7-11eb-8b44-e4187fe2ed3f.png)

Reply:
my github was being funky, i didn't see the messages u sent 4 hours ago till just now